src/librustc/diagnostics.rs

```diff
@@ -1886,7 +1886,51 @@ This explicitly states that you expect the trait object `SomeTrait` to
 contain references (with a maximum lifetime of `'a`).
 
 [1]: https://github.com/rust-lang/rfcs/pull/1156
-"##
+"##,
+
+E0454: r##"
```

Comment:
This is quite hard to say... It *can* be generalised, but like this, we have a very precise error code for each.